inc/Settings/Settings.php

```diff
@@ -101,14 +101,36 @@ public static function add_options_page(): void {
 	public static function settings_page_content(): void {
 		?>
 		<div id="vip-real-time-collaboration-settings-wrapper" class="wrap">
-			<h1><?php esc_html_e( 'VIP Real-Time Collaboration Settings', 'vip-real-time-collaboration' ); ?></h1>
+			<h1><?php esc_html_e( 'VIP Real-Time Collaboration', 'vip-real-time-collaboration' ); ?></h1>
+
+			<h2><?php esc_html_e( 'Plugin Settings', 'vip-real-time-collaboration' ); ?></h2>
 			<form action="options.php" method="post">
 				<?php
 				settings_fields( self::SETTINGS_PAGE_SLUG );
 				do_settings_sections( self::SETTINGS_PAGE_SLUG );
 				submit_button();
 				?>
 			</form>
+
+			<h2><?php esc_html_e( 'Plugin Debug Information', 'vip-real-time-collaboration' ); ?></h2>
```

Comment:
Since not all of this pertains to the plugin
```suggestion
			<h2><?php esc_html_e( 'Debug Information', 'vip-real-time-collaboration' ); ?></h2>
```